### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#634] [Announcements]
# https://plugin-comment2.vuejs.press/zh/guide/

设置选项 你既可以在 Node.js 一侧使用插件选项设置选项，也可以通过客户端配置文件在浏览器一侧设置选项。 tab 通过插件选项 tab 通过客户端配置文件 但是有以下你需要注意的限制： provider、多语言设置和其他资源相关选项必须在插件选项中设置。; 为确保 tree-shaking 有效，我们必须在 Node 一侧优化入口，以便打包器可以...

https://plugin-comment2.vuejs.press/zh/guide/

**Comment by zou88288** [comment]
了

**Reply by zou88288** [reply]
考虑

**Reply by zou88288** [reply]
接口
